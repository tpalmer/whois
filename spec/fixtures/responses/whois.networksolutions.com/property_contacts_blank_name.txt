NOTICE AND TERMS OF USE: You are not authorized to access or query our WHOIS
database through the use of high-volume, automated, electronic processes. The
Data in Network Solutions' WHOIS database is provided by Network Solutions for information
purposes only, and to assist persons in obtaining information about or related
to a domain name registration record. Network Solutions does not guarantee its accuracy.
By submitting a WHOIS query, you agree to abide by the following terms of use:
You agree that you may use this Data only for lawful purposes and that under no
circumstances will you use this Data to: (1) allow, enable, or otherwise support
the transmission of mass unsolicited, commercial advertising or solicitations
via e-mail, telephone, or facsimile; or (2) enable high volume, automated,
electronic processes that apply to Network Solutions (or its computer systems). The
compilation, repackaging, dissemination or other use of this Data is expressly
prohibited without the prior written consent of Network Solutions. You agree not to use
high-volume, automated, electronic processes to access or query the WHOIS
database. Network Solutions reserves the right to terminate your access to the WHOIS
database in its sole discretion, including without limitation, for excessive
querying of the WHOIS database or for failure to otherwise abide by this policy.
Network Solutions reserves the right to modify these terms at any time.

Get a FREE domain name registration, transfer, or renewal with any annual hosting package.

http://www.networksolutions.com

Visit AboutUs.org for more information about ALBENDERSKUNGFUACADEMY.COM
<a href="http://www.aboutus.org/ALBENDERSKUNGFUACADEMY.COM">AboutUs: ALBENDERSKUNGFUACADEMY.COM </a>




Registrant:
Al Bender's Kung Fu Academy of NJ
   260-262 Liberty Street
   Bloomfield, NJ 07003
   US

   Domain Name: ALBENDERSKUNGFUACADEMY.COM

   ------------------------------------------------------------------------
   Promote your business to millions of viewers for only $1 a month
   Learn how you can get an Enhanced Business Listing here for your domain name.
   Learn more at http://www.NetworkSolutions.com/
   ------------------------------------------------------------------------

   Administrative Contact:
      Owner		sifubender@AOL.COM
      Owner
      Al Bender's Kung Fu Academy of NJ
      260-262 Liberty Street
      Bloomfield, NJ 07003
      US
      999 999 9999 fax: 999 999 9999

   Technical Contact:
      		hostmaster@earthlink.net
      Earthlink Inc
      1430 West Peachtree St. NW, Ste. 400
      Atlanta, GA 30309
      US
      888-932-1997 fax: 123 123 1234


   Record expires on 18-Nov-2012.
   Record created on 18-Nov-1999.
   Database last updated on 4-Jul-2012 06:25:23 EDT.

   Domain servers in listed order:

   NS15.FASTDNSHOST.COM         96.9.177.101
   NS16.FASTDNSHOST.COM         96.9.177.102


