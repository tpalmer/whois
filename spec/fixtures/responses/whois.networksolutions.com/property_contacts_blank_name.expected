#technical_contacts
  should: %s CLASS(array)
  should: %s SIZE(1)
  should: %s[0] CLASS(contact)
  should: %s[0].type         == Whois::Record::Contact::TYPE_TECHNICAL
  should: %s[0].name         == ""
  should: %s[0].organization == "Earthlink Inc"
  should: %s[0].address      == "1430 West Peachtree St. NW, Ste. 400"
  should: %s[0].city         == "Atlanta"
  should: %s[0].zip          == "30309"
  should: %s[0].state        == "GA"
  should: %s[0].country_code == "US"
  should: %s[0].phone        == "888-932-1997"
  should: %s[0].fax          == "123 123 1234"
  should: %s[0].email        == "hostmaster@earthlink.net"